pkg/entrypoint/entrypointer_test.go

```diff
@@ -94,8 +99,8 @@ func TestEntrypointerFailures(t *testing.T) {
 
 func TestEntrypointer(t *testing.T) {
 	for _, c := range []struct {
-		desc, entrypoint, postFile string
-		waitFiles, args            []string
+		desc, entrypoint, postFile, terminationPath string
```

Comment:
can remove the `terminationPath` field now.

Reply:
done.